waku/v2/node/localnode.go

```diff
@@ -253,6 +279,11 @@ func (w *WakuNode) getENRAddresses(addrs []ma.Multiaddr) (extAddr *net.TCPAddr,
 		multiaddr = append(multiaddr, wssAddrs...)
```

Comment:
I am wondering if we append wss addresses, can any external node connect using it? Don't we need to have ceritficates setup in order for that to happen? Or is that only a browser restriction? I am not too clear on this

Reply:
if you dont have ssl setup then there is no point pushing the wss multiaddr.
This may only be useful when go-waku is used as a service node, which is out of scope.

Reply:
When you say service node, for which services? 
Because status desktop is acting as a  service node for filter, lightpush and peerexchange.

Reply:
I have seen this `wss` addresses being obtained automagically when using circuit relay. The node not only receives tcp addresses from the circuit relay server, but also, if that server supports wss, your node will also end up with a `wss` address. The connection will still happen because the `wss` in the multiaddress does not refer to your node, it refers to the circuit relay server.